sky/provision/instance_setup.py

```diff
@@ -214,6 +215,17 @@ def _setup_node(runner: command_runner.SSHCommandRunner,
                              ssh_credentials=ssh_credentials)
 
 
+def _ray_gpu_options(custom_resource: str) -> str:
```

Comment:
Add a 1-liner docstr? Is this a new fix not related to 2.9.3 upgrade?

Reply:
Yes, it seems ray 2.9.3 will not be able to detect the GPUs when launching with `--image-id docker:xxx` on GPU cluster.